aspnet/visual-studio/overview/2013/release-notes.md

```diff
@@ -154,15 +154,15 @@ This version of NuGet also removes the need to provide explicit consent for NuGe
 
 ### One ASP.NET
 
-The Web Forms project templates integrate seamlessly with the new One ASP.NET experience. You can add MVC and Web API support to your Web Forms project, and you can configure authentication using the One ASP.NET project creation wizard. For more information, see [Creating ASP.NET Web Projects in Visual Studio 2013](creating-web-projects-in-visual-studio.md).
+The Web Forms project templates integrate seamlessly with the new One ASP.NET experience. You can add MVC and Web API support to your Web Forms project, and you can configure authentication using the One ASP.NET project creation wizard. For more information, see Creating ASP.NET Web Projects in Visual Studio 2013.
```

Comment:
These are a tough call since that content collection no longer exists.   Some Visual Studio 2013 exists here: https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/h8w79z10(v=vs.120)

However, it is a small subset that does not contain the .NET/web development specific content.

I think it is best in this situation to not send the developer customer on a chase for document that does not exist.  So where it seems to be not disruptive to the information that is available on the page, deleting the reference to the non-existant content altogether in many of these cases seems like the only option.

@TDykstra, do you agree with the resolution just above?  Delete "For more information, see Creating ASP.NET Web Projects in Visual Studio 2013." since it does not exist?

Reply:
Agree